docs/topics/native/native-cocoapods.md

```diff
@@ -14,16 +14,117 @@ as well as [a Kotlin Gradle project and an Xcode project](native-cocoapods-xcode
 
 ## Set up an environment to work with CocoaPods
 
-Install the [CocoaPods dependency manager](https://cocoapods.org/):
+Install the [CocoaPods dependency manager](https://cocoapods.org/) using the installation tool of your choice:
+
+> If your device has an Apple M chip, follow our instructions [below](#apple-m-chips).
+>
+{type="note"}
+
+<tabs>
+<tab title="Homebrew">
+
+1. Install [Homebrew](https://brew.sh/) in case you don't have yet.
+2. Install Ruby. You can choose a specific version:
+
+    ```bash
+    brew install ruby@3.0
+    ```
+
+3. Install CocoaPods:
+
+    ```bash
+    brew install cocoapods
+    ```
+
+</tab>
+<tab title="RVM">
+
+1. Install [Ruby version manager](https://rvm.io/rvm/install) in case you don't have yet.
+2. Install Ruby. You can choose a specific version:
+
+    ```bash
+    rvm install ruby 3.0.0
+    ```
+
+3. Install CocoaPods:
+
+    ```bash
+    sudo gem install cocoapods
+    ```
+
+</tab>
+<tab title="Rbenv">
+
+1. Install [rbennv from GitHub](https://github.com/rbenv/rbenv#installation) in case you don't have yet.
+2. Install Ruby. You can choose a specific version:
+
+    ```bash
+    rbenv install 3.0.0
+    ```
+
+3. Set the Ruby version as local for a particular directory or global for the whole machine:
+
+    ```bash
+    rbenv global 3.0.0
+    ```
+
+4. Install CocoaPods:
+
+    ```bash
+    sudo gem install cocoapods
+    ```
+
+</tab>
+<tab title="Bundler">
```

Comment:
I'm not sure that we should promote using Bundler right now. After installation of CocoaPods via Bundler, it won't be possible to use `pod` from CocoaPods gradle plugin because it doesn't support execution of pods via `bundle exec pod install` our plugin just don't know anything about Bundler and his commands.